economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+
+\subsubsection{Note regarding `inflation rates'}
+
+Decentralized network monetary policy often makes reference to a so-called `inflation rate'. In this model, since Phase 1 sees an constant amount of tokens minted in each period, the effective rate at which the circulating supply grows will decrease period-to-period (and year-to-year). In Phase 2, the number of tokens minted per day decreases, which accelerates the decline in the supply growth rate. Stakes generally also experience a decreasing growth rate, the magnitude of which is heavily affected by discretionary stake configurations. To avoid misleading terminology, this paper does not refer to a rate, instead using \textit{issuance} as a short-hand for the absolute sum of tokens minted per period.
+
+\subsection{Rationale for two-phase model}
+
+\subsubsection{Demand uncertainty}\label{demand}
+
+These demand-related premises form the basis for the two-phase model: 
+\begin{enumerate}
+\item There is little certainty regarding the length of time that will elapse between the network's launch and the development of meaningful demand from users (developers, end-users \& others).
+\item This early-stage demand is particularly fragile, because new users are naturally more fickle. Specifically, neither developers nor their end-users have compelling reasons to tolerate an inadequate service (e.g. too few service-providers available to manage a sharing policy), such as the sunk costs of integration, reliance on the service, or other forms of path dependency, all of which are more likely to play a role later in the network's lifetime. Early users are unavoidably required to take a greater risk than later adopters, as they do not have the experience of existing customers as evidence of a reliable or valuable service.
+\item Many existing subsidy models involve daily token issuance dropping exponentially [ADD REFERENCE], right from launch. However, if the network's fee market does not reach a sufficient stage of maturity in time to make up for this sharp reduction in subsidy, some node operations may be rendered entirely unsustainable. 
+\item Most importantly, some early adopters will come to NuCypher with a use case that inflexibly requires long-lasting sharing policies. This means that the protocol must motivate the successive re-commitment to long token lock durations (i.e. staking for 12+ months) by a sufficient population of service-providers. A diminished subsidy, in lieu of a mature fee market, may not be enough to incentivize this repeated and essential re-commitment. 
```

Comment:
"early adopters [may come to NuCypher with] a use case" -> "early adopters may employ a use case"

```suggestion
\item Most importantly, some early adopters may employ a use case that inflexibly requires long-lasting sharing policies. This means that the protocol must motivate the successive re-commitment to long token lock durations (i.e. staking for 12+ months) by a sufficient population of service-providers. A diminished subsidy, in lieu of a mature fee market, may not be enough to incentivize this repeated and essential re-commitment. 
```

Reply:
```suggestion
\item Most importantly, there will be some early adopters of NuCypher whose use case inflexibly requires long-lasting sharing policies. This means that the protocol must motivate the successive re-commitment to long token lock durations (i.e. staking for 12+ months) by a sufficient population of service-providers. A diminished subsidy, in lieu of a mature fee market, may not be enough to incentivize this repeated and essential re-commitment. 
```